protocol-definitions/Topic.yaml

```diff
@@ -103,3 +104,83 @@ methods:
           since: 2.0
           doc: |
             True if registration is removed, false otherwise
+  - id: 4
+    name: publishAll
+    since: 2.0
+    doc: |
+      Publishes all messages to all subscribers of this topic
+    request:
+      retryable: false
+      partitionIdentifier: name
+      params:
+        - name: name
+          type: String
+          nullable: false
+          since: 2.0
+          doc: |
+            Name of the Topic
+        - name: messages
+          type: List_Data
+          nullable: false
+          since: 2.0
+          doc: |
+            The messages to publish to all subscribers of this topic
+    response: {}
+  - id: 5
+    name: publishAsync
+    since: 2.0
+    doc: |
+      Publishes the message to all subscribers of this topic
+    request:
+      retryable: false
+      partitionIdentifier: name
+      params:
+        - name: name
+          type: String
+          nullable: false
+          since: 2.0
+          doc: |
+            Name of the Topic
+        - name: message
+          type: Data
+          nullable: false
+          since: 2.0
+          doc: |
+            The message to publish to all subscribers of this topic
+    response:
+      params:
+        - name: response
+          type: long
+          nullable: false
+          since: 2.0
+          doc: |
+            the CompletionStage to synchronize on completion.
```

Comment:
@andrewoelfing 
I believe this is the registration id, not the CompletionStage , right ?
Also check response of `publishAllAsync`

Reply:
Hello @sancar, what do you mean with the registration id? There is no response when the event is published (please see `EventService#publishEvent`) . Maybe this can the number of currently published messages to the topic?

Reply:
`type: long` got me confused. There should not be any response. Can you make this part 
`response {}` as it is in publishAll/publish ? Same goes for `publishAllAsync` as well.

Reply:
Yes, I will change it.